api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -1350,6 +1350,33 @@ type AWSPlatformSpec struct {
 	// +kubebuilder:default=false
 	// +optional
 	MultiArch bool `json:"multiArch"`
+
+	// SharedVPC contains fields that must be specified if the HostedCluster is using a VPC that was
+	// created in a different AWS account and has been shared to the AWS account where the HostedCluster
+	// will be created.
```

Comment:
```suggestion
	// SharedVPC contains fields that must be specified if the HostedCluster must use a VPC that is
	// created in a different AWS account and is shared with the AWS account where the HostedCluster
	// will be created.
```

Is there a reason you used the present continuous tense in  `if the HostedCluster is using a VPC` , and later used a future tense `where the HostedCluster will be created` ? 

Fwiw, I made this suggestion to show how I would have written it.